### PR TITLE
[Import Group] Merge for ranking APIs; Update tables

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/place/golden/get_locations_ranking_test.go [hunk 1]
```diff
@@ -29,77 +29,86 @@ import (
...
-		if diff := cmp.Diff(response, &expected, protocmp.Transform()); diff != "" {
-			t.Errorf("payload got diff: %v", diff)
-			continue
+			if e2e.GenerateGolden {
+				e2e.UpdateProtoGolden(response, goldenPath, c.goldenFile)
+				continue
+			}
+
+			var expected pb.GetLocationsRankingsResponse
+			if err = e2e.ReadJSON(goldenPath, c.goldenFile, &expected); err != nil {
```
Comment (Spaceenter) [inline internal/server/place/golden/get_locations_ranking_test.go:104]: nit:

err := ...

File: internal/server/place/golden/get_related_locations_test.go [hunk 2]
```diff
@@ -31,64 +31,82 @@ import (
...
+			}
+			goldenFile := path.Join(goldenPath, c.goldenFile)
+			if e2e.GenerateGolden {
+				e2e.UpdateProtoGolden(resp, goldenPath, c.goldenFile)
+				continue
+			}
+
+			var expected pb.GetRelatedLocationsResponse
+			file, _ := ioutil.ReadFile(goldenFile)
+			err = protojson.Unmarshal(file, &expected)
```
Comment (Spaceenter) [inline internal/server/place/golden/get_related_locations_test.go:101]: nit:

if err := XXX; err != nil {

**Review by Spaceenter — APPROVED** [review]

File: internal/server/place/golden/get_locations_ranking_test.go (hunk 1 above)
Comment (shifucun) [reply internal/server/place/golden/get_locations_ranking_test.go]: Done

File: internal/server/place/golden/get_related_locations_test.go (hunk 2 above)
Comment (shifucun) [reply internal/server/place/golden/get_related_locations_test.go]: Done

File: internal/server/place/golden/get_related_locations/state.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "data": {
+    "CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase": {
```
Comment (pradh) [inline internal/server/place/golden/get_related_locations/state.json:3]: Wonder why this changes?  Is it because this merge happens for current branch too?

File: internal/server/place/places.go [hunk 4]
```diff
@@ -186,10 +190,15 @@ func GetRelatedLocations(
 		return nil, err
 	}
 	result := &pb.GetRelatedLocationsResponse{Data: map[string]*pb.RelatedPlacesInfo{}}
-	for statVar, data := range baseDataList[0] {
-		if data == nil {
-			result.Data[statVar] = nil
-		} else {
+	for _, baseData := range baseDataList {
```
Comment (pradh) [inline internal/server/place/places.go:193]: nit: Comment on merge strategy or rationale?

File: internal/server/place/places.go [hunk 5]
```diff
@@ -269,10 +278,15 @@ func GetLocationsRankings(
 	}
 
 	result := &pb.GetLocationsRankingsResponse{Data: map[string]*pb.RelatedPlacesInfo{}}
-	for statVar, data := range baseDataList[0] {
-		if data == nil {
-			result.Data[statVar] = nil
-		} else {
+	for _, baseData := range baseDataList {
```
Comment (pradh) [inline internal/server/place/places.go:281]: nit: Comment on merge strategy or rationale?

File: internal/server/place/golden/get_related_locations/state.json (hunk 3 above)
Comment (shifucun) [reply internal/server/place/golden/get_related_locations/state.json:3]: This is a new test added to cover merge case for dc_branch table
